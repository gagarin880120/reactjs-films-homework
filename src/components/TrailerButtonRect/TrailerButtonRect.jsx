import React from 'react';
import styles from './TrailerButtonRect.module.scss';
export default function TrailerButtonRect(props) {
  return (
  <button
    onClick={props.onTrailerButtonClick}
    className={styles.watchButton}
  >
    Watch Now
  </button>
  );
}

