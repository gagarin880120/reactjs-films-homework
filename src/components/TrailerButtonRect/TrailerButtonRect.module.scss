.watchButton {
  color: #fff;
  font-size: 16px;
  outline: 0;
  background: #09ade3;
  border: none;
  border-radius: 3px;
  width: 114px;
  height: 35px;
  cursor: pointer;
}

.watchButton:active {
  transform: translateY(2px);
}

@media screen and (max-width: 600px) {
  .watchButton {
    width: 70px;
  }
}
