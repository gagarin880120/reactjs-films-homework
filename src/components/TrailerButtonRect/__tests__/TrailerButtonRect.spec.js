import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import TrailerButtonRect from '../TrailerButtonRect';

test('TrailerButtonRect component should render without crashing', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<TrailerButtonRect />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
