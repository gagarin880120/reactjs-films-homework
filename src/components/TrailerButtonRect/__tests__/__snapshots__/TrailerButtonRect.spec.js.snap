// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrailerButtonRect component should render without crashing 1`] = `
<button
  className="trailerButton"
  onClick={[Function]}
  type="button"
>
  Watch Now
</button>
`;
